cicd/app/api_docs.yaml

```diff
@@ -123,7 +123,7 @@ Resources:
         MinTTL: 1
         Name: !Sub
                 - "api-docs-cache-Policy-${ResourceName}"
-                - ResourceName: !If [ IsDev, !Ref pCleanBranch, !Ref pEnvironment ]
```

Comment:
This is superfluous, the conditional could be removed but it's a special case for this repo that all dev environments share resources. In other repos they will be separately named